src/polkadot_rpc_wrapper.py

```diff
@@ -73,17 +73,19 @@ def get_system_peers(self) -> Tuple[list, bool]:
         peer_list = response_json['result']
         return peer_list, True
 
-    def has_session_key(self, session_key):
+    def has_session_key(self, session_key) -> Tuple[bool, str]:
         """
         Checks if this node has the supplied session_key (E.g. 0xb75f94a5eec... )
         :param session_key: string
-        :return: boolean
+        :return: Tuple[bool, str]
         """
         data = '{"id": 1, "jsonrpc":"2.0", "method": "author_hasSessionKeys", "params":["' + session_key + '"]}'
         response = requests.post(url=self.__server_address, headers=self.__headers, data=data)
         response_json = json.loads(response.text)
+        if 'error' in response_json.keys():
+            return False, response_json['error']['message']
         result = response_json['result']
-        return result
+        return result, ""
```

Comment:
I believe the Pythonic way to handle "returning" an error isn't to do it like this, but to raise an actual `Error` and then except it in the function above (or pass it on up, depending on what the aim is). I'm not saying we can't do it like this, especially with a cool tool like the package you linked, but there are other ways too.

```suggestion
    def has_session_key(self, session_key) -> bool:
        """
        Checks if this node has the supplied session_key (E.g. 0xb75f94a5eec... )
        :param session_key: string
        :return: bool
        """
        data = '{"id": 1, "jsonrpc":"2.0", "method": "author_hasSessionKeys", "params":["' + session_key + '"]}'
        response = requests.post(url=self.__server_address, headers=self.__headers, data=data)
        response_json = json.loads(response.text)
        if 'error' in response_json.keys():
            raise SomeCustomError(response_json['error']['message'])
        return response_json['result']
```

This comment applies to all functions where you added a tuple for errors.